src/main/java/com/mastercard/dis/mids/reference/MIDSReferenceApplication.java

```diff
@@ -207,7 +213,22 @@ void generateMultiSDKToken() {
     void performEnrollment(boolean claimSharingFlow) {
         try {
             log.info(claimSharingFlow ? "<<--- Claim Sharing Enrollment Started --->>" : "<<--- Enrollment Started --->>");
-            midsReference.performEnrollment(claimSharingFlow);
+
+            Cache.pdsEnrollment =midsReference.getPDS(claimSharingFlow, Collections.singletonList("attributePDS"));
```

Comment:
Cache.pdsEnrollment =midsReference.getPDS(claimSharingFlow, Collections.singletonList("attributePDS"));

Fix the white space after the '='